railties/lib/rails/generators/rails/app/templates/config/puma.rb.tt

```diff
@@ -8,6 +8,11 @@ max_threads_count = ENV.fetch("RAILS_MAX_THREADS") { 5 }
 min_threads_count = ENV.fetch("RAILS_MIN_THREADS") { max_threads_count }
 threads min_threads_count, max_threads_count
 
+# Specifies the `worker_timeout` threshold that Puma will use to wait before
+# terminating a worker in development environments.
+#
+worker_timeout 3600 if ENV.fetch("RAILS_ENV", "development") == "development"
```

Comment:
👋 since you don’t care whether `RAILS_ENV` is set or not, would it make sense to express it as something more concise?

```suggestion
worker_timeout 3600 if ENV["RAILS_ENV"] == "development"
```

Reply:
If `ENV["RAILS_ENV"]` is unset, the environment is development by default, and the setting should be enabled.